### DISCUSSION TITLE
[x][postponed][manuscript][outreach][blueprint] EGA experiment not in identifiers.org (was: " experiment:global_id")

### DISCUSSION THREAD

**Discussion by dzerbino** [fairtracks/fairtracks_standard#98] [Team Posts]
No infrastructure for this at the moment.

**Comment by sveinugu** [comment]
There is the GEO series concept: https://www.ncbi.nlm.nih.gov/geo/info/overview.html, for instance, and other possibilities. I think we should document possible solutions for this, including project-specific ones that are still registered in identifiers.org (like e.g. ENCODE). Not sure what will fit BLUEPRINT, but hopefully there is a identifiers.org-resolvable global experiment-like id available?

**Comment by sveinugu** [comment]
Should in any case not be required. But not optimal if we are unable to provide global ids for our example BLUEPRINT metadata..

**Comment by sveinugu** [comment]
Just to clarify. I don't aim for a single, global repository of experiments. Just a CURIE that points towards a stable description of the experiment, somehow.

**Comment by dzerbino** [comment]
But if we had a CURIE pointing to a description of the stable description of the experiment, wouldn't we have a _de facto_ registry of experiments?

**Comment by sveinugu** [comment]
Well in a sense yes. A registry of experiments that has tracks as an end output. But it would be a subset of the curies handled by identifiers.org, and the persistence of the records will be the responsibility of the hosting services. I don’t envision that we will be able to find experiment global ids at a level that directly matches the experiment as modeled in our standard. In my simple ENCODE-based example, the global_id of the experiment points to a GEO record of all ENCODE histone modification chip-seq carried out by the lab. The local_id is a more direct match, as this is the ENCODE experiment id, which happens to be resolvable by identifiers.org. However I decided to showcase GEO instead of just copying the local_id into global_id. The main point of the field, in my head, is to point outwards at a stable FAIR record where one could get more details and preferably be able to access the raw data.

**Comment by dzerbino** [comment]
I agree with the long term vision, but can this field be required at this point in time, if its implementation is still undefined?

**Comment by sveinugu** [comment]
I think it is very well defined now: Any identifiers.org-resolvable curie that points to a stable site with experiment information (preferably with links to the raw data). Anything like that in the BLUEPRINT metadata? The rest is just recommendations.

**Comment by dzerbino** [comment]
Right, but this definition implies an infrastructure which alas does not exist. We can generate fictitious IDs for Blueprint but for them to be identifiers.org-resolvable, we need a service that identifiers.org can redirect to.

**Comment by sveinugu** [comment]
But isn’t the raw data available somewhere like GEO or ArrayExpress or simliar? Or is it only in a BLUEPRINT portal which is then not registered in identifiers.org? Or perhaps EGA?  Or nowhere? But perhaps the granularity of such deposits are only on the study level? Perhaps showcasing this lack is then actually a nice output from this IS?

**Comment by jmfernandez** [comment]
BLUEPRINT raw data is registered in EGA. You can find the correspondences in the files hosted here: ftp://ftp.ebi.ac.uk/pub/databases/blueprint/ega_dataset/

On 6/6/19 11:34, sveinugu wrote:
>
> But isn’t the raw data available somewhere like GEO or ArrayExpress or simliar? Or is it only in a BLUEPRINT portal which is then not registered in identifiers.org? Or perhaps EGA? Or nowhere? But perhaps the granularity of such deposits are only on the study level? Perhaps showcasing this lack is then actually a nice output from this IS?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/22/comments/9?email_source=notifications&email_token=AAPZ4VRGK252ANPKSGVEKUDPZDK2HA5CNFSM4HRFKBP2YY3PNVWWK3TUL52HS4DFWNCGS43DOVZXG2LPNZIG643UKJSXA3DZVJRW63LNMVXHIX3JMTHAAAPOHE>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAPZ4VX3EKTZHOPSL72IC5TPZDK2HANCNFSM4HRFKBPQ>.
>
-- 

"There is no reason why anybody would want a computer in their home" -
	Ken Olson, founder of DEC 1977
"640K ought to be enough for anybody" - Bill Gates, 1981 
"Nobody will ever outgrow a 20Mb hard drive." - ???

"Premature optimization is the root of all evil." - Donald Knuth
"Los ordenadores son inútiles. Sólo pueden darte respuestas" - Pablo Ruíz Picasso

José María Fernández González
Senior Research Scientist
e-mail: jose.m.fernandez@bsc.es
INB Node, Life Sciences Department
Torre Girona Building, 1st floor, Barcelona Supercomputing Center
C/. Jordi Girona, 31
Zip Code: 08034				Barcelona (Spain)
Phone: (+34) 934117074

**Comment by dzerbino** [comment]
OK, I can put in EGAX identifiers, but to be clear they won't be resolvable by identifiers.org (EGA experiments are not a recognised namespace) and in future this will be a problem for non-sequencing based experiments (e.g. analysis runs)

**Comment by sveinugu** [comment]
Strange, only EGAS and EGAD CURIEs are recognized by identifiers.org, sample and datasets, not the other EGA accession number types. However, we are involved with the Local EGA project so perhaps we can use this connection to solve the issue. In any case, seems the correct thing to do. 

I think we agreed in some other discussion point below that global_id should not be a requirement, but highly recommended, so if it is a problem, one can skip it (but preferably with a bit of shame!..) :)

**Comment by dzerbino** [comment]
Note: I discovered belatedly that although Blueprint data is in EGA, they have ENA IDs, and thus experiments, samples etc can be access via the ena.embl namespace... I'm rather confused by this, but I'm sure there's a good explanation.

**Comment by dzerbino** [comment]
Hmm, this is odd the study IDs are valid (e.g. [ERA211714](https://www.ebi.ac.uk/ena/data/view/ERA211714)) but the experiment IDs are not (e.g. [ERX547919](https://www.ebi.ac.uk/ena/data/search?query=ERX547919))... the plot thickens...

**Comment by sveinugu** [comment]
@jmfernandez How does the EGAX ids currently validate towards identifiers.org?

**Comment by jmfernandez** [comment]
Still no news from our EGA contacts in Barcelona, sorry

**Comment by sveinugu** [comment]
Closed the duplicate discussion: https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/32

**Comment by sveinugu** [comment]
Linked to from https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/41 and https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/49. Postponing
